docs/clipboard-unsanitized/explainer.md

```diff
@@ -185,8 +185,11 @@ EndFragment:<start offset of the end fragment comment tag>
 ```
 
 ### Read()
-Follow the algorithm specified in [read()](https://w3c.github.io/clipboard-apis/#dom-clipboard-read) except for the below step:
-1. If `text/html` representation is present in the [ClipboardItem](https://w3c.github.io/clipboard-apis/#clipboard-item-interface) and `text/html` is present in the `unsanitized` list, then return the blobData as-is without any sanitization.
+Follow the algorithm specified in [read()](https://w3c.github.io/clipboard-apis/#dom-clipboard-read) except for the below steps:
+1. If `text/html` representation is present in the [ClipboardItem](https://w3c.github.io/clipboard-apis/#clipboard-item-interface) and `text/html` is present in the `unsanitized` list, then follow the below steps:
+    1. If size of `unsanitized` list is greater than 1, then throw `Support to read multiple unsanitized formats is not implemented.` exception.
```

Comment:
Are there any exceptions that say things like (ie. variations of "not yet implemented")? Not supported sounds reasonable for exceptions.